nvidia-merged/PKGBUILD

```diff
@@ -290,6 +291,8 @@ package_nvidia-merged-utils() {
     install -D -m644 "${_vgpupkg}/init-scripts/systemd/nvidia-vgpu-mgr.service" "${pkgdir}/usr/lib/systemd/system/nvidia-vgpu-mgr.service"
     install -D -m644 "${srcdir}/vgpu_unlock-rs.conf" "${pkgdir}/usr/lib/systemd/system/nvidia-vgpud.service.d/30-vgpu_unlock-rs.conf"
     install -D -m644 "${srcdir}/vgpu_unlock-rs.conf" "${pkgdir}/usr/lib/systemd/system/nvidia-vgpu-mgr.service.d/30-vgpu_unlock-rs.conf"
+    install -D -m644 "${srcdir}/unlock-profile_override.toml" "${pkgdir}/etc/vgpu_unlock/profile_override.toml"
```

Comment:
Well I have seen many users have problem with resolution, that's why I thought it could be helpful.
Users will at least have a config template that they could work on.

Reply:
You can't simply ship a single hard-coded configuration. If a user edits this and then tries to update the package, it will fail.

If an example is necessary, there needs to be a specific sample file drawn up and included that details all the available options. That file, however, cannot be written to `/etc/vgpu_unlock/profile_override.toml`.

Again, though, this is not relevant to live migration support.